evm/utils/fixture_tests.py

```diff
@@ -284,7 +284,7 @@ def normalize_transactiontest_fixture(fixture):
     try:
         normalized_fixture['rlp'] = decode_hex(fixture['rlp'])
     except binascii.Error:
-        normalized_fixture['rlpHex'] = fixture['rlp']
```

Comment:
So this is actually intentional (and likely important) so that there is a clear and unambiguous differentiation between RLP that has been decoded to bytes from it's hex form, and RLP that failed decoding.